src/routes/SheltersPege/SheltersPege.js

```diff
@@ -0,0 +1,187 @@
+    console.log("Calling renderPets")
+    // no pets exist
+    if (pets.length === 0) {
+      return (
+        <div className='alert alert-success' role='alert'>
+          More pets will be available soon! Check back later :)
+        </div>
+      );
+    } else {
+      return (
+        <div>
+          <Section>
+            {this.renderFilter()}
+          </Section>
+          <Section>
+            <div className='row'>
+              {pets.map(pet => {
+                let typeOfAnimal = (pet.TypeOfAnimal === "Cat" || pet.TypeOfAnimal === "Dog") ? pet.TypeOfAnimal : "Other"
+
+                let petAge = ""
+                if (parseInt(pet.Age) <= 1 && typeOfAnimal === "Cat") petAge = "Kitten"
+                else if (parseInt(pet.Age) <= 1 && typeOfAnimal === "Dog") petAge = "Puppy"
+                else if (parseInt(pet.Age) <= 2) petAge = "Young"
+                else if (parseInt(pet.Age) <= 7) petAge = "Adult"
+                else petAge = "Senior"
+
+                let cName = "hideCard"
+                if (typeOfPets === "Any" && age === "Any" && available === "Any") cName = "showCard"
+                if (typeOfPets === "Any" && age === "Any" && available === pet.Availability) cName = "showCard"
+                if (typeOfPets === "Any" && age === petAge && available === pet.Availability) cName = "showCard"
+                if (typeOfPets === typeOfAnimal && age === "Any" && available === "Any") cName = "showCard"
+                if (typeOfPets === typeOfAnimal && age === petAge && available === "Any") cName = "showCard"
+                if (typeOfPets === "Any" && age === petAge && available === "Any") cName = "showCard"
+                if (typeOfPets === "Any" && age === petAge && available === pet.Availability) cName = "showCard"
+                if (typeOfPets === typeOfAnimal && age === "Any" && available === pet.Availability) cName = "showCard"
+                if (typeOfPets === typeOfAnimal && age === petAge && available === pet.Availability) cName = "showCard"
+
+                return this.renderPet(pet, cName)
+              })}
+            </div>
+          </Section>
+
+        </div>
+
+      )
+    }
+  }
+
+  renderFilter() {
+    console.log("Calling renderFilter")
+    return (
+      <FormGroup className="flex">
+        <div className="select-wrap" style={{ width: "100%" }}>
+          <label>Type of Pets</label>
+          <select id='typeOfPets' name="typeOfPets" onChange={e => this.inputChanged('typeOfPets', e.target.value)}>
+            <option value='Any'>Any</option>
+            <option value='Cat'>Cat</option>
+            <option value='Dog'>Dog</option>
+            <option value='Other'>Other</option>
+          </select>
+        </div>
+
+        <div className="select-wrap" style={{ width: "100%" }}>
+          <label>Age</label>
+          <select id='age' name="age" onChange={e => this.inputChanged('age', e.target.value)}>
+            <option value='Any'>Any</option>
```

Comment:
That's pretty cool! I tried it in dev and it looks like this.
![image](https://user-images.githubusercontent.com/71732016/153672347-10dc7280-e6b9-464f-8cc8-ffe6e90b2a07.png)
I like this slider but it might be easier for a user to select from the six categories. If a user select age==4, the page will only show age==4 pets, but is we use "Adult", it will show the pets with age 3-4. Let me know what you think!

Reply:
I guess I meant two sliders, one for min and one for max. That way the user can select their own age range. This could also be done with two number inputs either instead of or in addition to sliders. I though there was a way to put two numbers on a single slider (for min and max) but I think I was wrong about that

Reply:
ah! I see. yeah that would be nice. I'm sure that there is a way to do that. I will keep it in mind and add it if I have free time in the future.

Reply:
Sorry to be a pain, but I think this should be changed before this PR is approved. I think a user-defined age range is better UX than a dropdown. Plus, not all animals have the same lifespan. 3 years is young for a dog, but old for a rat. It doesn't need to be anything crazy, just two number inputs to define the min/max ages

Reply:
Per our conversation, I will prioritize image uploader first, then work on this fix. I will create a bug fix card so that I won't forget. Thank you!